docs/tutorials/prowler-app-security-hub-integration.md

```diff
@@ -0,0 +1,244 @@
+# AWS Security Hub Integration
+
+**Prowler App** enables automatic export of security findings to AWS Security Hub, providing seamless integration with AWS's native security and compliance service. This comprehensive guide demonstrates configuration and management of AWS Security Hub integrations to centralize security findings and enhance compliance tracking across AWS environments.
+
+When enabled and configured, scan results are automatically sent to AWS Security Hub after each scan completes. Findings are formatted in ASFF (AWS Security Finding Format), enabling native integration with AWS security workflows and compliance frameworks.
+
+???+ note
+    AWS Security Hub must be enabled in target regions before configuring this integration. Refer to [AWS Security Hub pricing](https://aws.amazon.com/security-hub/pricing/) for cost information.
+
+## Prerequisites
+
+Before configuring the AWS Security Hub Integration in **Prowler App**, ensure the following requirements are met:
+
+### AWS Security Hub Setup
+
+1. **Enable AWS Security Hub** in at least one AWS region for the target AWS account
+2. **Accept the Prowler Integration** in AWS Security Hub to allow findings import
+3. **Configure IAM Permissions** for `securityhub:BatchImportFindings` and `securityhub:GetFindings`
+
+???+ info "AWS Configuration Required"
+    AWS Security Hub must be properly configured before setting up the integration in **Prowler App**. For detailed instructions on enabling Security Hub and accepting the Prowler integration, refer to the [AWS Security Hub Setup Guide](./aws/securityhub.md).
+
+???+ warning
+    If no region has Security Hub enabled, the integration in **Prowler App** will not work. Security Hub is a regional service - enable it in each region where findings should be received.
+
+## Required Permissions
+
+### IAM Policy
+
+The Security Hub integration requires specific permissions to export findings. The IAM role or user credentials must include:
+
+- `securityhub:BatchImportFindings` - To send findings to Security Hub
+- `securityhub:GetFindings` - To retrieve existing findings for updates
```

Comment:
Keep this.

Reply:
Since I suggest to remove the entire section, I think we should keep it in another section.